Tests/TuistCacheIntegrationTests/ContentHashing/ContentHashingIntegrationTests.swift

```diff
@@ -44,7 +45,11 @@ final class ContentHashingIntegrationTests: TuistTestCase {
         } catch {
             XCTFail("Error while creating files for stub project")
         }
-        subject = CacheGraphContentHasher(contentHasher: CacheContentHasher())
+        let mockXcodeBuildController = MockXcodeBuildController()
+        mockXcodeBuildController.versionStub = {
+            Observable.just(.standardOutput("Xcode 13.0\nBuild version 13A233"))
```

Comment:
There's content here that is not relevant for the caching. Therefore I recommend extracting the hasheable information and hashing that instead:

```swift
stuct XcodeVersion {
  let version: Version
  let buildVersion: Int
}
```

Reply:
I see, but we would add a dependency from the output of that command, and from the hashing point of view I don't think we get any benefit (hash will change when either version or build version changes)

Reply:
Even if it's the same behavior, it's not correct. We need to hash raw information because the formatting doesn't add value to it.

Reply:
I see the point of hashing raw information when possible.
However, in this case to extract the raw information we would need to add a strong dependency from the `version` output format, which might change in future release, and I'm not sure the tradeoff is worth it in this case

Reply:
If instead we think it's worth it, which part of the swift version you think it's relevant? is the "pure" swift version (e.g 5.4.2) enough?
```
Apple Swift version 5.4.2 (swiftlang-1205.0.28.2 clang-1205.0.19.57)
```

Reply:
regarding the dependency from the output, using 5.5 already gives a different format:
```
swift-driver version: 1.26.9 Apple Swift version 5.5 (swiftlang-1300.0.31.1 clang-1300.0.29.1)
```

Reply:
Refactored to use `System.swiftVersion`, so now we are only considering the actual swift version 🚀